### ISSUE TITLE
Make breadcrumbs elements height at least 24px

### ISSUE THREAD

**Issue by kkoryaka** [Office-of-Digital-Services/California-State-Web-Template-Development#154]
Elements should be at least 24 by 24 pixels (a11y requirements)
